src/main/java/com/monumental/services/ModelService.java

```diff
@@ -281,6 +281,39 @@ public void delete(List<Integer> ids) throws HibernateException {
         return records;
     }
 
+    /**
+     * Generic method to build a "FROM ... WHERE ..." query
+     * Assumes the FROM clause is the name of the T generic class
+     * @param whereParameterName - Name of the column to use in the WHERE clause
+     * @param whereParameter - Used to set the parameter in the WHERE clause
+     * @return List<T> - the List of T returned by the query
+     */
+    @SuppressWarnings("unchecked")
+    public List<T> getFromWhere(String whereParameterName, Object whereParameter) {
```

Comment:
This should also have an option to initialize all the lazy loaded lists like in the other gets

Reply:
Check out hibernate `Criteria` objects https://docs.jboss.org/hibernate/orm/3.5/javadocs/org/hibernate/Criteria.html

Instead of having this function be limited to one equals, you could instead have `ModelService.createCriteria`, then you do all your `.add`s to it and pass the completed `Criteria` object to a `ModelService.executeQuery(Criteria criteria)`